CMakeLists.txt

```diff
@@ -120,42 +79,46 @@ install(TARGETS ${POCLIB}
   INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
 )
 
-configure_package_config_file(${POCLIB}Config.cmake.in
+configure_package_config_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/cmake/${POCLIB}Config.cmake.in
   ${CMAKE_CURRENT_BINARY_DIR}/${POCLIB}Config.cmake
   INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${POCLIB}
+  NO_SET_AND_CHECK_MACRO # poc does not provide legacy style define
 )
 
 # ...and the version file
 write_basic_package_version_file(
   ${CMAKE_CURRENT_BINARY_DIR}/${POCLIB}ConfigVersion.cmake
   VERSION ${PORTS_OF_CALL_VERSION}
-  COMPATIBILITY SameMajorVersion )
+  COMPATIBILITY SameMajorVersion
+)
 
 # Install the cmake configuration files
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${POCLIB}Config.cmake
               ${CMAKE_CURRENT_BINARY_DIR}/${POCLIB}ConfigVersion.cmake
-        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${POCLIB} )
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${POCLIB}
+)
 
 # Install the source header files
-install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/ports-of-call/portability.hpp"
-              "${CMAKE_CURRENT_SOURCE_DIR}/ports-of-call/portable_arrays.hpp"
+install(
+  FILES "${CMAKE_CURRENT_SOURCE_DIR}/ports-of-call/portability.hpp"
+        "${CMAKE_CURRENT_SOURCE_DIR}/ports-of-call/portable_arrays.hpp"
   DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${POCLIB}
-  )
+)
 
 # Install the export target. This will define the CMake target
 # for external projects when used with `find_package`
-install(EXPORT ${POCLIB}Targets
+install(
+  EXPORT ${POCLIB}Targets
   NAMESPACE ${POCLIB}::
-  FILE "${POCLIB}Targets.cmake"
-  DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${POCLIB}"
-  COMPONENT dev)
+  FILE ${POCLIB}Targets.cmake
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${POCLIB}
+)
 
-# Export configuration for external projects that reference
-# just our build-tree; e.g. for submodules. To use, ensure 
-# `CMAKE_PREFIX_PATH` points to this source directory.
 # NOTE: This config will not be relocatable!
-export(TARGETS ${POCLIB} NAMESPACE ${POCLIB}::
+export(
+  TARGETS ${POCLIB} 
+  NAMESPACE ${POCLIB}::
   FILE "${CMAKE_CURRENT_BINARY_DIR}/${POCLIB}Targets.cmake")
-export(PACKAGE ${POCLIB})
```

Comment:
Is this no longer necessary?

Reply:
It wasn't really necessary to begin with, and in fact on it's own does nothing for `cmake@3.15+`. See https://cmake.org/cmake/help/latest/command/export.html#exporting-packages.

I believe the inclusion was originally my fault, as I misunderstood what `export(PACKAGE ...)` was doing. In previous versions of `cmake` (prior to `3.15`), it did naughty things like stealthy dumping lots of cmake configs into a hidden `~/.cmake` dir (much like spack) that changed the behavior of cmake's `find_package()` in frustrating ways.